play-json/shared/src/test/scala/play/api/libs/json/MacroSpec.scala

```diff
@@ -27,29 +25,25 @@ object TestFormats {
     } yield a -> b
   }
 
-  implicit def tuple2OWrites[A: Writes, B: Writes]: OWrites[(A, B)] =
-    OWrites {
-      case (a, b) => Json.obj("_1" -> a, "_2" -> b)
-    }
+  implicit def tuple2OWrites[A: Writes, B: Writes]: OWrites[(A, B)] = OWrites {
+    case (a, b) => Json.obj("_1" -> a, "_2" -> b)
+  }
 }
 
 final class TextId(val value: String) extends AnyVal
 
-import org.scalatest._
 import org.scalacheck.Gen
 
 class MacroSpec extends AnyWordSpec with Matchers with org.scalatestplus.scalacheck.ScalaCheckPropertyChecks {
+  import MacroSpec._
+
   "Reads" should {
     "be generated for simple case class" in {
       val json     = Json.obj("bar" -> "lorem")
       val expected = Simple("lorem")
 
       forAll(
-        Gen.oneOf(
-          Json.reads[Simple],
-          Json.configured.reads[Simple],
```

Comment:
Feature no longer available?

Reply:
Default arguments (and therefore MacroOptions) aren't supported in Scala 3, so I dropped them here, but I kept the original in MacroScala2Spec.

Reply:
Not sure to follow, with default arguments? `Json.configured` won't be available in Scala3? Naming and option handling no longer configurable in this way?

Reply:
`Json.configured.parse` is available.  `Json.configured.stringify` is available.  Et cetera.  Just not `Json.configured.reads`/writes/format - because those are macros and they're not re-implementable in Scala 3's typeclass derivation mechanisms.